pmm/pmm2-release.groovy

```diff
@@ -504,7 +504,6 @@ ENDSSH
                         declare -A repos=(
                             ["percona-grafana"]="percona-platform/grafana"
                             ["percona-dashboards"]="percona/grafana-dashboards"
-                            ["percona-qan-api2"]="percona/qan-api2"
```

Comment:
Not sure about what this line is responsible for. Looks like we're dropping a repo but don't provide any replacement.

Reply:
It creates a tag on those repos. In this case it's already done on line 508.